content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -867,9 +868,10 @@ To deploy a Windows container on Kubernetes, you must first create an example ap
     * Service discovery, `curl` the service name with the Kubernetes [default DNS suffix](/docs/concepts/services-networking/dns-pod-service/#services)
     * Inbound connectivity, `curl` the NodePort from the Linux master or machines outside of the cluster
     * Outbound connectivity, `curl` external IPs from inside the pod using kubectl exec
-    {{< note >}}
-    **Windows _container hosts_ are not able to access the IP of services scheduled on them due to current platform limitations of the Windows networking stack. Only Windows _pods_ are able to access service IPs.
-    {{< /note >}}
+
+{{< note >}}
```

Comment:
- If you want the page to look like the other content pages, you can add a template type attribute in the front matter:
`content_template: templates/concept`. With this template, you are required to include an `overview` and then a `body`. See another page under concepts for an example. This template will provide an internal toc at the top, based upon the main headers in the doc. May help navigation of the page.
- Or, I see that other pages in setup do not set a template type and instead include the toc shortcode at the top,
{{< toc >}}.  See content/en/docs/setup/node-conformance.md as an example.



- nit: the tables could be converted to Markdown

Reply:
let us merge this PR and i will create a new one to do these changes. very helpful hints @kbhawkey

Reply:
@kbhawkey i fixed this in another PR and will ping you there :)